src/Main.elm

```diff
@@ -0,0 +1,82 @@
+module Main exposing (init, main, update, view)
+
+import Browser
+import Element exposing (centerX, centerY, el, fill, height, html, layout, map, maximum, padding, rgb255, shrink, spacing, text, width)
+import Element.Background as Background
+import Element.Border as Border
+import Element.Font exposing (center)
+import Element.Input exposing (button)
+import Ranking
+import Setup
+
+
+main : Program () Model Msg
+main =
+    Browser.sandbox
+        { init = init
+        , view = layout [] << view
+        , update = update
+        }
+
+
+type Msg
+    = SetupMsg Setup.Msg
+    | RankingMsg Ranking.Msg
+    | SwitchView
+
+
+type alias Model =
+    { setupModel : Setup.Model
+    , rankingModel : Ranking.Model
+    , switchButton : Bool
+    }
```

Comment:
`switchButton` isn't a very indicative name and the 'Model' suffix isn't really needed.
How about:
```elm
type alias Model =
    { setup : Setup.Model
    , ranking : Ranking.Model
    , screen : CurrentScreen
    }
```
?

Reply:
Hmm..
Yeah, you're right.